tools/scalajsenv.js

```diff
@@ -125,6 +125,8 @@ ScalaJS.clz32 = ScalaJS.g["Math"]["clz32"] || (function(i) {
 // Other fields
 //!if outputMode == ECMAScript51Global
 ScalaJS.d = {};         // Data for types
+ScalaJS.a = {};         // Scala.js-defined JS class value accessors
+ScalaJS.b = {};         // Scala.js-defined JS class value fields
 ScalaJS.c = {};         // Scala.js constructors
```

Comment:
Just do double check: There are three of these, such that during initialization of a SJS defined class, the class itself is not yet available externally.